src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/OffenderPrimaryIdentifiers.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.justice.digital.delius.jpa.standard.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(toBuilder = true)
+@Entity
+@Table(name = "OFFENDER")
+public class OffenderPrimaryIdentifiers {
```

Comment:
This means findAll will only select and distinct the few columns we are interested in, e.g exclude  CLOBs etc